templates/problem/list.html

```diff
@@ -233,7 +233,8 @@ <h3>{{ _('Hot problems') }} <i class="fa fa-fire"></i></h3>
                             <a href="{{ sort_links.ac_rate }}">{{ _('AC %%') }}{{ sort_order.ac_rate }}</a>
                         </th>
                         <th class="editorial">
-                            {{ _('Editorial') }}
+                            <a href="{{ sort_links.editorial }}"><i class="fa fa-book"></i>{{ sort_order.editorial }}
```

Comment:
```suggestion
                            <a href="{{ sort_links.editorial }}">
                                <i class="fa fa-book"></i>{{ sort_order.editorial }}
```

Reply:
Fixed, and also updated the formatting of the place a few lines above from which I copied it.